src/main/java/net/sf/jabref/logic/importer/fileformat/BibtexParser.java

```diff
@@ -119,6 +124,16 @@ public static ParserResult parse(Reader in, ImportFormatPreferences importFormat
         return Optional.of(entries.iterator().next());
     }
 
+    @Override
+    public List<BibEntry> parseEntries(InputStream inputStream) throws ParserException {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream, StandardCharsets.UTF_8));
```

Comment:
On the one hand, I think, it's good to have `UTF-8` as default encoding. On the other hand, there are preferences where the user sets the default encoding.

Maybe we should remove the global preference and always set UTF-8 as default encoding?